qiskit_experiments/characterization/t1_experiment.py

```diff
@@ -0,0 +1,169 @@
+
+        fit_out, fit_err, fit_cov, chisq = curve_fit_wrapper(
+            exp_fit_fun,
+            delays,
+            means,
+            stddevs,
+            p0=[amplitude_guess, t1_guess, offset_guess],
+        )
+
+        analysis_result = AnalysisResult(
+            {
+                "value": fit_out[1],
+                "stderr": fit_err[1],
+                "unit": experiment_data._data[0]["metadata"]["unit"],
+                "label": "T1",
+                "fit": {
+                    "params": fit_out,
+                    "stderr": fit_err,
+                    "labels": ["amplitude", "T1", "offset"],
+                    "chisq": chisq,
+                    "cov": fit_cov,
+                },
+            }
+        )
+
+        #pylint: disable = too-many-boolean-expressions
+        if (
+            abs(fit_out[0] - 1.0) < 0.1
+            and abs(fit_out[2]) < 0.1
+            and chisq < 3
+            and (fit_err[0] is None or fit_err[0] < 0.1)
+            and (fit_err[1] is None or fit_err[1] < fit_out[1])
+            and (fit_err[2] is None or fit_err[2] < 0.1)
+        ):
+            analysis_result["quality"] = "computer_good"
+        else:
+            analysis_result["quality"] = "computer_bad"
+
+        return analysis_result, None
+
+
+class T1Experiment(BaseExperiment):
+    """T1 experiment class"""
+
+    __analysis_class__ = T1Analysis
+
+    def __init__(self, qubit: int, delays: Union[List[float], np.array], unit: str = "dt"):
+        """
+        Initialize the T1 experiment class
+
+        Args:
+            qubit: the qubit whose T1 is to be estimated
+            delays: delay times of the experiments
+            unit: unit of the duration. Supported units: 's', 'ms', 'us', 'ns', 'ps', 'dt'.
+                Default is ``dt``, i.e. integer time unit depending on the target backend.
+
+        Raises:
+            ValueError: if the number of delays is smaller than 3
+        """
+        if len(delays) < 3:
+            raise ValueError("T1 experiment: number of delays must be at least 3")
+
+        self._delays = delays
+        self._unit = unit
+        super().__init__([qubit], type(self).__name__)
+
+    # pylint: disable=arguments-differ
+    def circuits(self, backend: Optional["Backend"] = None) -> List[QuantumCircuit]:
+        """
+        Return a list of experiment circuits
+
+        Args:
+            backend: Optional, a backend object
+
+        Returns:
+            The experiment circuits
+        """
+
+        circuits = []
+
+        for delay in self._delays:
+            circ = QuantumCircuit(1, 1)
+            circ.x(0)
+            circ.barrier(0)
+            circ.delay(delay, 0, self._unit)
+            circ.barrier(0)
+            circ.measure(0, 0)
+
+            # pylint: disable = eval-used
+            circ.metadata = {
+                "experiment_type": self._type,
+                "qubit": self.physical_qubits[0],
```

Comment:
You could just generically leave this as 
```suggestion
                "qubits": self.physical_qubits,
```

Reply:
I guess I'll do it once we have a standard metadata. Otherwise, if this experiment has only one qubit, then what's the advantage of this suggestion?

Reply:
Is there a plan to have some form of standardized metadata which is defined somewhere? Or at least some required fields?

Reply:
A partial answer to this question: https://github.com/Qiskit/qiskit-experiments/issues/12